specs.json

```
@@ -452,6 +452,10 @@
     "url": "https://webassembly.github.io/exception-handling/js-api/",
     "forkOf": "wasm-js-api-2"
   },
+  {
+    "url": "https://webassembly.github.io/gc/core/",
+    "forkOf": "wasm-core-1"
```

```suggestion
    "forkOf": "wasm-core-2"
```
it seems to be a fork of level 2 - but then for some reason, it looks like we don't have level 2 in specs.json - @tidoust, does that ring a bell?

---

See https://github.com/w3c/browser-specs/pull/593 but we should revisit that. I created #1059 accordingly.